applications/asset_tracker/README.rst

```diff
@@ -52,19 +57,22 @@ Requirements
 User interface
 **************
 
-The two buttons and two switches are used to enter a pairing pattern to associate a specific board with an nRF Cloud user account.
-In addition, the two switches have the following function when the connection is established:
+The buttons and switches have the following functions when the connection is established:
+
+Button 1:
+    * Send a BUTTON event to nRF Cloud.
+    * Toggle GPS function ( long press the button for a minimum of 10 seconds)
```

Comment:
Suggestion: `Toggle GPS operation`
Also, an extra space seems to be present after `(`

Reply:
Implemented